panic: runtime error: invalid memory address or nil pointer dereference

It seems that there are pull requests included that have not yet been merged.

We need to investigate why the `FetchPullRequestNumbers` function returned the numbers of pull requests that have not been merged yet.

Error:
```
git-pr-release-go: version: 0.1.3, commit: b192a9d3f6cf8e7c857a2336e627d32aeac4a1e4, date: 2024-08-12T15:28:52Z
git-pr-release-go: Found pull requests:  [1 2 3]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6f96d6]
```

Code:
https://github.com/odanado/git-pr-release-go/blob/v0.1.3/github_client.go#L81-L83

--- a/github_client.go
+++ b/github_client.go
@@ -75,7 +75,9 @@ func (c *GithubClient) FetchPullRequests(ctx context.Context, prNumbers []int) (
 			return nil, err
 		}
 
-		pullRequests = append(pullRequests, *pr)
+		if pr.MergedAt != nil {
+			pullRequests = append(pullRequests, *pr)
+		}
 	}
 
 	slices.SortFunc(pullRequests, func(a, b github.PullRequest) int {

